src/components/HelloWorld.vue

```diff
@@ -0,0 +1,207 @@
+<template>
+  <v-container fluid pa-0>
+    <v-layout
+      row
+      wrap
+      justify-center
+    >
+      <v-flex xs12 style="background-color: #FFDA48" class="text-xs-center">
+        <v-img
+          :src="require('../assets/canarilogo.png')"
+          contain
+          height="200"
+        >
+        </v-img>
+        <h1 class="display-2 font-weight-bold mb-3 mt-2 pa-3">
+          Canary Islands
+          <br>
+          Developers Community
+        </h1>
+        <p class="subheading">YA SOMOS MÁS DE {{ computedTotalFirms }} PERSONAS!</p>
+      </v-flex>
+      <v-flex
+        xs12
+        mb-5
+      >
+        <v-layout justify-center mt-2>
+          <a
+            v-for="(eco, i) in ecosystem"
+            :key="i"
+            :href="eco.href"
+            class="subheading mx-3"
+            target="_blank"
+          >
+            {{ eco.text }}
+          </a>
+        </v-layout>
+      </v-flex>
+      <v-flex xs12 sm8 class="pa-3">
+        <h3 class="display-2 mb-4">Nuestro manifiesto</h3>
+        <p class="subheading">Somos un grupo de profesionales de las Islas Canarias del sector de la programación con diverso grado de experiencia. Entendemos Canarias como un conjunto, rechazando la hegemonía de una isla sobre las demás.</p>
+        <p class="subheading">El grupo se forma desde el sentimiento generalizado de que la comunidad de desarrolladores y desarrolladoras de software de Canarias no está localmente valorada como merece y esto tiene un impacto en la reducción de oportunidades laborales dignas en las islas.</p>
+        <p class="subheading">Los profesionales y las profesionales que trabajamos en esta comunidad hemos demostrado en reiteradas ocasiones estar al mismo nivel que nuestros colegas de otras nacionalidades y nuestro trabajo genera uno de los más altos valores competitivos para cualquier empresa.</p>
+        <p class="subheading">Generando comunidad, visibilizando mejores oportunidades y fomentando el diálogo entre profesionales podremos cambiar las cosas.</p>
+        <p class="subheading">Es por ello que:</p>
+        <ul class="mb-4">
+          <li
+           v-for="(point, index) in criticalPoints"
+           :key="index"
+           class="subheading"
+          >
+            {{ point }}
+          </li>
+        </ul>
+        <p class="subheading">Cualquier persona que participe en el ciclo de desarrollo del software es bienvenida a unirse, escuchar y participar. Somos una comunidad inclusiva y respetuosa con todos los géneros, identidades y formas de expresión.</p>
+        <h3 class="display-2 mb-4">Libro de Firmas</h3>
+        <p class="subheading">Con nuestras firmas declaramos estar de acuerdo con el manifiesto y colaborar en la medida de nuestras posibilidades a hacerlo una realidad.</p>
+        <p class="subheading">{{ totalFirms }} personas ya han firmado el manifiesto.</p>
```

Comment:
Embedding the text in a vue template might make it harder to edit the manifest, should we keep it as markdown?

Reply:
{{ totalFirms }} and {{ computedTotalFirms }} are the real-time updated values of the total of members, I can't think of how to create a reactivity linked to the database with MD

Reply:
Yeah, absolutely, I mean the rest of the text actually... :-)